### DISCUSSION TITLE
Webhook Message Open

### DISCUSSION THREAD

**Discussion by jshapiro2** [postalserver/postal#2551] [Development discussions]
Hello,

Is there a way to have postal send webhook when a message is opened?

**Comment by willpower232** [comment]
If you activate click tracking for your domain then yes

https://docs.postalserver.io/features/click-and-open-tracking/
https://docs.postalserver.io/developer/webhooks/#message-click-event

**Reply by jshapiro2** [reply]
Thanks but that is not what I am asking.  Currently, if someone clicks postal will send a payload via webhook.  If someone opens, postal tracks the open from inside the web app but does not send a payload via webhook.

**Reply by willpower232** [reply]
Ah my mistake, the open event is called loaded

![image](https://github.com/postalserver/postal/assets/1619102/94c8eaa3-6322-4d59-acee-38451e30656b)
